pyproject.toml

```diff
@@ -286,3 +285,349 @@ lint.ignore = [
 "jwst/ramp_fitting/tests/*" = [
     "E402", # module-level import not at top of file
 ]
+
+[tool.towncrier]
+filename = "CHANGES.rst"
+directory = "changes"
+package = "jwst"
+title_format = "{version} ({project_date})"
+ignore = [".gitkeep"]
+wrap = true
+issue_format = "`#{issue} <https://github.com/spacetelescope/jwst/issues/{issue}>`_"
+
+[[tool.towncrier.type]]
+directory = "general"
+name = "General"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "docs"
+name = "Documentation"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "stpipe"
+name = '``stpipe``'
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "datamodels"
+name = "Data Models"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "associations"
+name = "Associations"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "scripts"
+name = "Scripts"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "fits_generator"
+name = "``fits_generator``"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "set_telescope_pointing"
+name = '``set_telescope_pointing``'
+showcontent = true
+
+# steps
+
+[[tool.towncrier.type]]
+directory = "align_refs"
+name = 'align_refs (``coron3``)'
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "ami"
+name = 'ami_analyze / ``ami_normalize`` / ``ami_average`` (``ami3``)'
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "assign_mtwcs"
+name = 'assign_mtwcs (``image3``, ``spec3``)'
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "assign_wcs"
+name = 'assign_wcs (``image2``, ``spec2``)'
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "background"
+name = 'background (``image2``, ``spec2``)'
+showcontent = true
+
```

Comment:
Suggest adding an entry for `badpix_selfcal` step, used in spec2.

Reply:
```suggestion

[[tool.towncrier.type]]
directory = "badpix_selfcal"
name = 'badpix_selfcal (``spec2``)'
showcontent = true

```